UI/views/schedule.html

```diff
@@ -27,9 +27,9 @@ <h3 class="colHeader"><span class="sr-only">Schedule</span>
          </div>
       </div>
       <div class="alert alert-info" ng-show="!filteredItems.length  && !loadingSchedule">
-      <p ng-if="scheduleStatus.ctools.status !==200 && scheduleStatus.canvas.status !==200">CTools and Canvas unreachable.</p>
-      <span ng-show="(scheduleStatus.ctools.status ==200 || scheduleStatus.canvas.status ==200) && schedule.length!==0">{{showWhen | noItemsFoundForTime}}</span>
-      <span ng-show="schedule.length===0 && (scheduleStatus.ctools.status ===200 || scheduleStatus.canvas.status ===200)">No items due in CTools or Canvas.</span></div>
+        <span ng-show="schedule.length !==0">{{showWhen | noItemsFoundForTime}}</span>
+        <span ng-show="schedule.length ===0">No items due in CTools or Canvas.</span>
```

Comment:
The outer condition is !filteredItems.length - that is the filtered time slice you are looking at contains no items. This could be because there are no items for any time, or because there are no items in the time slice you are looking at.

length === 0 no items in any of the feeds
length !== 0 there are items in feeds

So if no items: "No items due in CTools or Canvas"
There are items: but none for this slice (outer condition): "Slice appropriate message about no items"